.eslintrc

```diff
@@ -13,7 +13,29 @@
     "react"
   ],
   "ecmaFeatures": {
-    "jsx": true
+    "jsx": true,
+    "arrowFunctions": true,
+    "binaryLiterals": true,
+    "blockBindings": true,
+    "classes": true,
+    "defaultParams": true,
+    "destructuring": true,
+    "forOf": true,
+    "generators": true,
```

Comment:
if it's not going to be allowed then blocking it in the linting for now seems sensible, it would avoid any confusion